py/tests/test_processing.py

```diff
@@ -325,6 +325,22 @@ def test_validate_project_config():
     assert str(e.value) == 'json atom at path ".foobar" is missing from rhs'
 
 
+def test_validate_cardinality_limit_config():
```

Comment:
This test validates a valid and an invalid config. Let's keep them in separate unit tests.

Reply:
Yeah, the whole things was copy paste from `validate_project_config` 🙈